### ISSUE TITLE
Video and Webcam input support

### ISSUE THREAD

**Issue by tenkaisynapse** [stevensona/shader-toy#75]
Is there a way to make iChannels use video files or webcam inputs as they do on the web based iChannel?

**Comment by Malacath-92** [comment]
No, it's not currently supported. I'd wager a guess that it will be difficult or impossible to do in VSCode, but it's nothing we investigated yet.

**Comment by Malacath-92** [comment]
See https://github.com/microsoft/vscode/issues/54097
It is pretty unlikely that we will support videos as input because of this.

**Comment by b005t3r** [comment]
Has anything about this changed in the last 4 years maybe? Having video input would be a great feature.
